truffle/com.oracle.truffle.api.debug/src/com/oracle/truffle/api/debug/Debugger.java

```diff
@@ -274,8 +274,8 @@ public Breakpoint setTagBreakpoint(int ignoreCount, com.oracle.truffle.api.instr
      * </ul>
      */
     @TruffleBoundary
-    void prepareContinue(int depth) {
-        getCurrentDebugContext().setAction(depth, new Continue());
+    void prepareContinue(DebugExecutionContext context, int depth) {
```

Comment:
Indeed, I think package-private is sufficient here. The original `private` won't suffice anymore however. Will change that. Thanks.